docs/operation/backup.md

```diff
@@ -0,0 +1,73 @@
+# Backup Data Using BanyanDB Backup Tool
+
+The backup command-line tool is used to backup BanyanDB snapshots to remote storage. The tool allows both one-time and scheduled backups.
+
+## Overview
+
+The backup tool performs the following operations:
+
+- Connects to a BanyanDB data node via gRPC to request the latest snapshots.
+- Determines the local snapshot directories based on the catalog type:
```

Comment:
So, this tool will be deployed as a sidecar(in k8s) of a data node and request through grpc locally?

Reply:
If so, I think we should add some docs about this tool should be used in the same place of the data node.

Reply:
> So, this tool will be deployed as a sidecar(in k8s) of a data node and request through grpc locally?

Or, it will be included into DB server binary folder and just run locally?

Reply:
Added in the `Prerequisites` section

Reply:
> > So, this tool will be deployed as a sidecar(in k8s) of a data node and request through grpc locally?
> 
> Or, it will be included into DB server binary folder and just run locally

My preference is to build a separate binary as `bydbbackup` which contains two commands: `backup` and `restore`. We can assemble the binary into the current image.